network-configuration/advanced-network-configuration/src/main/java/MemberA.java

```diff
@@ -0,0 +1,106 @@
+import com.hazelcast.config.Config;
+import com.hazelcast.config.EndpointConfig;
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.config.ServerSocketEndpointConfig;
+import com.hazelcast.config.WanPublisherConfig;
+import com.hazelcast.config.WanReplicationConfig;
+import com.hazelcast.config.WanReplicationRef;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import java.util.Collections;
+
+public class MemberA {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+
+        // AdvancedNetworkConfig works similarly to NetworkConfig however
+        // allows finer control of network configuration
+        // In order to use AdvancedNetworkConfig, it must be explicitly enabled
+        config.getAdvancedNetworkConfig().setEnabled(true);
+
+        // configure cluster joiner: disable default multicast joiner, enable & configure TCP/IP
+        config.getAdvancedNetworkConfig().getJoin().getMulticastConfig().setEnabled(false);
+        config.getAdvancedNetworkConfig().getJoin()
+              .getTcpIpConfig().setEnabled(true).addMember("192.168.1.10,192.168.1.11,192.168.1.12");
+
+        // configure the MEMBER protocol server socket to listen on 192.168.1.0-255:5701
+        config.getAdvancedNetworkConfig().setMemberEndpointConfig(createMemberEndpointConfig());
+
+        // configure the CLIENT protocol server socket:
+        // - listen on 10.10.200.10-12
+        // - advertise public address 172.10.10.10:9000
+        config.getAdvancedNetworkConfig().setClientEndpointConfig(createClientEndpointConfig());
+
+        // configure outgoing WAN connections to use SSL
+        config.getAdvancedNetworkConfig().addWanEndpointConfig(createWanEndpointConfig());
+
+        // setup WAN replication
+        config.addWanReplicationConfig(
+                new WanReplicationConfig()
+                        .setName("active-wan-replication")
+                        .addWanPublisherConfig(
+                            new WanPublisherConfig()
```

Comment:
Actually, connection to `147.102.1.10:8443` is not really configured on Cluster-A side. I think it can be useful to configure it here.